app/src/features/apiClient/screens/apiClient/components/clientView/components/Collection/components/CollectionsVariablesView/CollectionsVariablesView.tsx

```diff
@@ -33,6 +34,10 @@ export const CollectionsVariablesView: React.FC<CollectionsVariablesViewProps> =
     return setCollectionVariables(pendingVariables, collection.id)
       .then(() => {
         toast.success("Variables updated successfully");
+        trackVariablesUpdated({
+          type: "collection_variable",
+          updated_variable_count: pendingVariables.length,
+        });
```

Comment:
Please refer [here](https://github.com/requestly/requestly/pull/2701#discussion_r1961502790)

Event to be changes to `variables_saved` with params `num_variables`  and `type`